framework/src/org/checkerframework/framework/type/typeannotator/IrrelevantTypeAnnotator.java

```diff
@@ -0,0 +1,113 @@
+package org.checkerframework.framework.type.typeannotator;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.type.PrimitiveType;
+import javax.lang.model.type.TypeKind;
+import javax.lang.model.type.TypeMirror;
+import javax.lang.model.util.Types;
+import org.checkerframework.framework.type.AnnotatedTypeFactory;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedExecutableType;
+import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedWildcardType;
+import org.checkerframework.javacutil.TypesUtils;
+
+public class IrrelevantTypeAnnotator extends TypeAnnotator {
```

Comment:
Some javadoc would be good, in particular that the set of classes that the parameter takes are the relevant classes and that this Annotator does not annotate those types.